validator/src/admin_rpc_service.rs

```diff
@@ -303,6 +327,111 @@ impl AdminRpc for AdminRpcImpl {
         AdminRpcImpl::set_identity_keypair(meta, identity_keypair, require_tower)
     }
 
+    fn set_block_engine_config(
```

Comment:
how about valdiators wanting to reverse this?

Reply:
you mean set the config to nothing?

Reply:
you mean set the config to nothing?